src/comm/aca_grpc.cpp

```diff
@@ -94,6 +104,22 @@ Status GoalStateProvisionerImpl::ShutDownServer()
 
 void GoalStateProvisionerImpl::RunServer()
 {
+  ACA_LOG_INFO("%s\n", "Trying to init a new sub to connect to the NCM");
+  grpc::ChannelArguments args;
+  // Channel does a keep alive ping every 10 seconds;
+  args.SetInt(GRPC_ARG_KEEPALIVE_TIME_MS, 10000);
+  // If the channel does receive the keep alive ping result in 20 seconds, it closes the connection
+  args.SetInt(GRPC_ARG_KEEPALIVE_TIMEOUT_MS, 20000);
+
+  // Allow keep alive ping even if there are no calls in flight
```

Comment:
this is good change, we will want to think about how to handle the connection closing after the current POC, it will involve reconnect/retry etc.